rsp_vision/save/tables_manager.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+import pandas as pd
+
+
+class AnalysisSuccessTable:
+    def __init__(self, path: Path):
+        self.path = path / "analysis_success.log"
+
+        if not self.path.exists():
+            self.df = pd.DataFrame(
+                columns=["dataset_name", "date", "latest_job_id", "error"]
+            )
+            self.df.to_csv(self.path)
+        else:
+            self.df = pd.read_csv(self.path, index_col=0, header=0)
+
+    def update(
```

Comment:
Out if interest under what circumstances is this dataframe typically updated? with information on a new subject / session or for different rounds of analysis? I will drop by to see the code in action